tomcat/src/main/java/org/apache/coyote/http11/frontcontroller/FrontController.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.coyote.http11.frontcontroller;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.Objects;
+import nextstep.jwp.controller.Controller;
+import nextstep.jwp.controller.ControllerAdvice;
+import nextstep.jwp.exception.notfound.ControllerNotFoundException;
+import org.apache.coyote.http11.httpmessage.request.HttpRequest;
+import org.apache.coyote.http11.httpmessage.response.HttpResponse;
+import org.apache.coyote.http11.requestmapping.ApiHandlerMapper;
+import org.apache.coyote.http11.requestmapping.FileHandlerMapper;
+import org.apache.coyote.http11.requestmapping.RequestMapper;
+
+public class FrontController {
+
+    private static final List<RequestMapper> HANDLER_MAPPERS = List.of(new FileHandlerMapper(), new ApiHandlerMapper());
+    private static final ControllerAdvice CONTROLLER_ADVICE = new ControllerAdvice();
+
+    public static void service(HttpRequest httpRequest, HttpResponse httpResponse) throws IOException {
+        try {
+            Controller controller = getController(httpRequest);
+            controller.service(httpRequest, httpResponse);
+        } catch (Exception exception) {
+            Exception targetException = getTargetException(exception);
+            CONTROLLER_ADVICE.handleException(httpResponse, targetException);
+        }
+        httpResponse.write();
+    }
```

Comment:
```suggestion
    public static void service(HttpRequest httpRequest, HttpResponse httpResponse) {
        try {
            Controller controller = getController(httpRequest);
            controller.service(httpRequest, httpResponse);
        } catch (Exception exception) {
            Exception targetException = getTargetException(exception);
            CONTROLLER_ADVICE.handleException(httpResponse, targetException);
        }
        httpResponse.write();
    }
```
IOException이 터질 부분이 없어보입니다!

Reply:
제거 했습니다! 754310f5002f410f05be1eb64a4e43e5cd7845de